src/frontend/themes.scss

```diff
@@ -425,14 +425,6 @@ body.nord-light {
     color: var(--navbar-active);
   }
 
-  body.nord-light .downIcon circle {
-    fill: var(--primary);
-  }
-
-  body.nord-light .currentDownload {
-    --success: #0bd48c;
-  }
-
```

Comment:
these were doing nothing, the selector is wrong (it ends up as `body.nord-light body.nord-light ...`, and there's selectors for this working a few lines above added last month)